src/main/java/org/folio/linked/data/configuration/ValidatorsConfiguration.java

```diff
@@ -0,0 +1,24 @@
+package org.folio.linked.data.configuration;
+
+import static org.folio.linked.data.util.Constants.Validators.VALIDATOR_CHAIN_RESOURCE_SAVE;
+import static org.folio.linked.data.util.Constants.Validators.VALIDATOR_CHAIN_RESOURCE_UPDATE;
+
+import org.folio.linked.data.domain.dto.ResourceRequestDto;
+import org.folio.linked.data.service.validation.LdValidator;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class ValidatorsConfiguration {
+
+  @Bean(VALIDATOR_CHAIN_RESOURCE_SAVE)
+  public LdValidator<ResourceRequestDto> resourceSaveValidatorsChain(LdValidator<ResourceRequestDto> lccnValidator) {
+    // in future task we can filter out disabled validators here
+    return lccnValidator;
+  }
+
+  @Bean(VALIDATOR_CHAIN_RESOURCE_UPDATE)
```

Comment:
I think we do not need two separate validator chains for save an update now. May be we can start with one validator (for both save and update) and create separate validator beans if a need arise in future.

Reply:
we'll probably need separate validators for LCCN duplication validation, we should validate LCCN for duplicate only on create right?

Reply:
done